### PR TITLE
Correct the wrong words in the prompt information

### PR DESCRIPTION
**Pull request by lit2430**
# Proposed changes

Correct the wrong words in the prompt information

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: manager/dm-server/src/main/java/org/apache/doris/stack/agent/AgentHeatbeatRunner.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ private void heartbeatCheck() {
                 agent.setStatus(AgentStatus.STOP);
                 agentRepository.save(agent);
                 agentCache.putAgent(agent);
-                log.warn("agent {} is unhealthly", agent.getHost());
+                log.warn("agent {} is unhealthily", agent.getHost());
```
Comment (EmmyMiao87) [inline manager/dm-server/src/main/java/org/apache/doris/stack/agent/AgentHeatbeatRunner.java:79]:
```suggestion
                log.warn("agent {} is unhealthy", agent.getHost());
```

**Review by EmmyMiao87 — CHANGES_REQUESTED** [review]

File: manager/dm-server/src/main/java/org/apache/doris/stack/agent/AgentHeatbeatRunner.java (hunk 1 above)
Comment (liteng2430) [reply manager/dm-server/src/main/java/org/apache/doris/stack/agent/AgentHeatbeatRunner.java:79]: ok，thank you

Comment (liteng2430) [reply manager/dm-server/src/main/java/org/apache/doris/stack/agent/AgentHeatbeatRunner.java]: already modify "unhealthily" to "unhealthy"

**Comment by jackwener** [comment]
`be/src/olap/olap_common.h`

`UNKNOW OLAP_FIELD_TYPE_STRING` can be replaced with `UNKNOWN ... `

**Review by yangzhg — APPROVED** [review]
LGTM

**Comment by Abhishekthakur2545** [comment]
is this issue still open

**Review by hf200012 — APPROVED** [review]
LGTM

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

**Pull request merged**